osbenchmark/utils/io.py

```diff
@@ -344,6 +370,18 @@ def _do_decompress_manually_with_lib(target_directory, filename, compressed_file
         compressed_file.close()
 
 
+def _do_decompress_zstd(target_directory, filename):
```

Comment:
Instead of creating a separate function for this, is it possible to implement this through the `_do_decompress_manually` as is done with other decompressors like `bz2`? This would remove boilerplate code

Reply:
Yes I'll be working on that, thanks